src/REopt.jl

```diff
@@ -64,9 +64,6 @@ using ArchGDAL
 using Roots: fzero  # for IRR
 global hdl = nothing
 
-const M3_TO_GAL = 264.172  # [gal/m^3]
```

Comment:
`const`s should be defined at the module level since they are global variables (and this way the order of the include statements does not impact which files can access these global variables). Please put these `const`s back here and move any other `const`s that you are aware of to this file.

Reply:
Thanks, I was wondering if I should have consolidated them all here instead.